lingua_franca/lang/parse_cs.py

```diff
@@ -1582,66 +1606,104 @@ def _text_cs_inflection_normalize(word, arg):
         word [Word]
 
     """
-    if arg ==1: # _extract_whole_number_with_text_cs
-    #Number one (jedna)
-        if len(word)==5 and word.startswith("jed"):
+    if arg == 1:  # _extract_whole_number_with_text_cs
+        # Number one (jedna)
+        if len(word) == 5 and word.startswith("jed"):
             suffix = 'en', 'no', 'ny'
-            if word.endswith(suffix,3):
-                word="jedna"
+            if word.endswith(suffix, 3):
+                word = "jedna"
 
-        #Number two (dva)
+        # Number two (dva)
         elif word == "dvě":
             word = "dva"
 
-    elif arg == 2: # extract_datetime_cs  TODO: This is ugly
-        if word == "hodina": word = "hodin"
-        if word == "hodiny": word = "hodin"
-        if word == "hodinu": word = "hodin"
-        if word == "minuta": word = "minut"
-        if word == "minuty": word = "minut"
-        if word == "minutu": word = "minut"
-        if word == "minutu": word = "minut"
-        if word == "sekunda": word = "sekund"
-        if word == "sekundy": word = "sekund"
-        if word == "sekundu": word = "sekund"  
-        if word == "dní": word = "den" 
-        if word == "dnů": word = "den" 
-        if word == "dny": word = "den"
-        if word == "týdny": word = "týden"
-        if word == "týdnů": word = "týden"
-        if word == "měsíců": word = "měsíc"
-        if word == "měsíce": word = "měsíc"
-        if word == "měsíci": word = "měsíc"
-        if word == "roky": word = "rok"
-        if word == "roků": word = "rok"
-        if word == "let": word = "rok"
-        if word == "včerejšku": word = "včera"
-        if word == "zítřku": word = "zítra"
-        if word == "zítřejší": word = "zítra"
-        if word == "ranní": word = "ráno"
-        if word == "dopolední": word ="dopoledne"
-        if word == "polední": word ="poledne"
-        if word == "odpolední": word = "odpoledne"
-        if word == "večerní": word = "večer"
-        if word == "noční": word = "noc"
-        if word == "víkendech": word = "víkend"
-        if word == "víkendu": word = "víkend"
-        if word == "všedních": word = "všední"
-        if word == "všedním": word = "všední"
-
-        #Months
-        if word == "únoru": word = "únor"
-        elif word == "červenci": word = "červenec"
-        elif word == "července": word = "červenec"
-        elif word == "listopadu": word = "listopad"
-        elif word == "prosinci": word = "prosinec"
+    elif arg == 2:  # extract_datetime_cs  TODO: This is ugly
```

Comment:
Agreed =P and could be reworded. I like TODOs because my editor has a TODO sidebar and highlighting. If others find them annoying, though, I respect that.

Reply:
I see what you mean! This is @Tony763's TODO, not from this PR. It appears in one of my commits because of the standardization pass; Czech was added after Jarbas' initial round.

Reply:
Nope I'm a fan of them too, it's the kind of thing that can get fixed up the next time someone is working on that file.